src/test/java/com/ericsson/ei/handlers/test/RmqHandlerTest.java

```diff
@@ -74,7 +74,7 @@ public void initProperties() {
         rmqHandler.setPort(port);
         rmqHandler.setDomainId(domainId);
         rmqHandler.setComponentName(componentName);
-        rmqHandler.setBindingKey(bindingKey);
+        rmqHandler.setBindingKeys(bindingKey);
```

Comment:
I guess bindingKey variable also to can be refactored to plural name.

Reply:
In this test its techincally only a single key but I can change it.